src/odemis/acq/align/test/fastem_test.py

```diff
@@ -53,6 +53,13 @@ class TestFastEMCalibration(unittest.TestCase):
 
     @classmethod
     def setUpClass(cls):
+        try:
+            import fastem_calibrations
+        except ImportError as err:
+            raise unittest.SkipTest(
+                f"Skipping the fastem tests, correct libraries to perform the tests are not available.\n"
+                f"Got the error: %s" % err)
```

Comment:
Can you also add this at the top of `odemis.acq.test.fastem_test.TestFastEMAcquisitionTask.test_pre_calibrate`?